com.unity.formats.fbx/Tests/FbxTests/ModelExporterTest.cs

```diff
@@ -852,7 +850,40 @@ public void TestBlendShapeExport(string fbxPath)
             Assert.That (fbxPath, Is.Not.Null);
 
             SkinnedMeshRenderer originalSMR, exportedSMR;
-            var exportedFbxPath = ExportSkinnedMesh (fbxPath, out originalSMR, out exportedSMR);
+            SetImportSettings setImportSettings = (importer) =>
+            {
+#if UNITY_2019_1_OR_NEWER
+                importer.importBlendShapes = true;
+                importer.optimizeMeshPolygons = false;
+                importer.optimizeMeshVertices = false;
+                importer.meshCompression = ModelImporterMeshCompression.Off;
+
+                importer.importNormals = ModelImporterNormals.Import;
+                importer.importTangents = ModelImporterTangents.CalculateMikk;
+
+                // If either blendshape normals are imported or weldVertices is turned off (or both),
+                // the vertex count between the original and exported meshes does not match.
+                // TODO (UT-3410): investigate why the original and exported blendshape normals split the vertices differently.
+                importer.importBlendShapeNormals = ModelImporterNormals.None;
+                importer.weldVertices = true;
+#else
+                importer.importBlendShapes = true;
+                importer.optimizeMesh = false;
+                importer.meshCompression = ModelImporterMeshCompression.Off;
+
+                // In 2018.3, the vertices still do not match unless no normals
+                // are imported.
+                importer.importNormals = ModelImporterNormals.None;
```

Comment:
in 2018.3 for blendshapes it is not sufficient to not import blendshape normals. There are discrepancies with the vertices if any normals are imported. (To be investigated with UT-3410 as well)